src/test/java/RestAssuredTests/tests/APITests.java

```diff
@@ -0,0 +1,95 @@
+package RestAssuredTests.tests;
+
+import org.testng.annotations.Test;
+
+import com.google.gson.Gson;
+
+import io.restassured.path.json.JsonPath;
+
+import org.testng.annotations.BeforeTest;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
+
+
+public class APITests {
+  @Test
+  public void verifiesUser2IsJanet() {
+	  given()
+	  .when()
+	  	.get("https://reqres.in/api/users/2")
+	  .then()
+	  	.assertThat()
+	  	.body("data.email", equalTo("janet.weaver@reqres.in"));
+  }
+  
+  @Test
+  public void verifiesCanLogin() {
```

Comment:
This test requires the POST http verb. Response will be 200, the test should also verify the response content ("token": "QpwL5tke4Pnpja7X4")